recipes-bsp/trusted-firmware-a/trusted-firmware-a.bb

```diff
@@ -0,0 +1,96 @@
+DESCRIPTION = "Trusted Firmware-A for Renesas RZ"
+
+require include/rzg2l-optee-config.inc
+inherit deploy
+
+PACKAGE_ARCH = "${MACHINE_ARCH}"
+
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302 \
+"
+
+S = "${WORKDIR}/git"
+
+SRC_URI:rzg2l-sbc = " \
+    git://github.com/Renesas-SST/rz-atf.git;name=machine;branch=${BRANCH};protocol=https \
+"
+
+BRANCH:rzg2l-sbc = "scarthgap/rz-sbc"
+SRCREV_machine:rzg2l-sbc = "${AUTOREV}"
+PV = "v2.9+git"
+
+PLATFORM:rzg2l-sbc = "g2l"
+EXTRA_FLAGS:rzg2l-sbc = "BOARD=sbc_1"
+PMIC_EXTRA_FLAGS:rzg2l-sbc = "BOARD=smarc_pmic_2"
+FLASH_ADDRESS_BL2_BP:rzg2l-sbc = "00000"
+FLASH_ADDRESS_FIP:rzg2l-sbc = "1D200"
+
+PMIC_BUILD_DIR = "${S}/build_pmic"
+
+FILES:${PN} = "/boot "
+SYSROOT_DIRS += "/boot"
+
+SEC_FLAGS = " \
+    ${@oe.utils.conditional("ENABLE_SPD_OPTEE", "1", " SPD=opteed", "",d)} \
+"
+EXTRA_FLAGS:append = "${SEC_FLAGS}"
+PMIC_EXTRA_FLAGS:append = "${SEC_FLAGS}"
+
+FILESEXTRAPATHS:append := "${THISDIR}/files"
+
+ECC_FLAGS = " DDR_ECC_ENABLE=1 "
+ECC_FLAGS += "${@oe.utils.conditional("ECC_MODE", "ERR_DETECT", "DDR_ECC_DETECT=1", "",d)}"
+ECC_FLAGS += "${@oe.utils.conditional("ECC_MODE", "ERR_DETECT_CORRECT", "DDR_ECC_DETECT_CORRECT=1", "",d)}"
+EXTRA_FLAGS:append = "${@oe.utils.conditional("USE_ECC", "1", " ${ECC_FLAGS} ", "",d)}"
+PMIC_EXTRA_FLAGS:append = "${@oe.utils.conditional("USE_ECC", "1", " ${ECC_FLAGS} ", "",d)}"
+
+# requires CROSS_COMPILE set by hand as there is no configure script
+export CROSS_COMPILE="${TARGET_PREFIX}"
+
+# Let the Makefile handle setting up the CFLAGS and LDFLAGS as it is a standalone application
+CFLAGS[unexport] = "1"
+LDFLAGS[unexport] = "1"
+AS[unexport] = "1"
+LD[unexport] = "1"
+
+do_compile() {
+    oe_runmake PLAT=${PLATFORM} ${EXTRA_FLAGS} bl2 bl31
+
+    if [ "${PMIC_SUPPORT}" = "1" ]; then
```

Comment:
I removed the statement related to PMIC because it's not supported on RZ/G2L-SBC board
Update in the commit: https://github.com/Renesas-SST/meta-renesas/commit/d949b337eceba29ab3cd27395a91a7f1eda1e8cc